### ISSUE TITLE
[Mono] Cannot compile with Mono

### ISSUE THREAD

**Issue by Teashrock** [godotengine/godot#34547]
**Godot version:**
3.2.beta4.official

**OS/device including version:**
Windows 10 x64

**Issue description:**
My attempt to compile Godot with Mono from source results in this:
```
"C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-beta4\modules\mono\editor\GodotTools\GodotTools.sln" (целе
вой объект по умолчанию) (1) ->
"C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-beta4\modules\mono\editor\GodotTools\GodotTools\GodotTools
.csproj" (целевой объект по умолчанию) (4) ->
(Целевой объект CoreCompile) ->
  Utils\OS.cs(8,7): error CS0246: Не удалось найти тип или имя пространства имен "Mono" (возможно, отсутствует директив
а using или ссылка на сборку). [C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-beta4\modules\mono\editor\G
odotTools\GodotTools\GodotTools.csproj]

    Предупреждений: 1
    Ошибок: 1

Прошло времени 00:00:05.22
scons: *** [bin\GodotSharp\Tools\GodotTools.dll] RuntimeError : 'msbuild' exited with error code: 1
Traceback (most recent call last):
  File "C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-beta4\modules\mono\build_scripts\solution_builder.py", line 150, in run_command
    subprocess.check_call(args, env=env_override)
  File "c:\windows\python37-32\Lib\subprocess.py", line 347, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\MSBuild\\Current\\Bin\\MSBuild.exe', 'C:\\Users\\Teashrock\\Documents\\Development\\Not-Mine\\godot-3.2-beta4\\modules\\mono\\editor/GodotTools/GodotTools.sln', '/p:Configuration=Release']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\teashrock\envs\py3.7.4\lib\site-packages\scons\SCons\Action.py", line 1209, in execute
    result = self.execfunction(target=target, source=rsources, env=env)
  File "C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-beta4\modules\mono\build_scripts\godot_tools_build.py", line 18, in build_godot_tools
    build_solution(env, solution_path, build_config)
  File "C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-beta4\modules\mono\build_scripts\solution_builder.py", line 214, in build_solution
    run_command(msbuild_path, msbuild_args, env_override=msbuild_env, name='msbuild')
  File "C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-beta4\modules\mono\build_scripts\solution_builder.py", line 152, in run_command
    raise RuntimeError("'%s' exited with error code: %s" % (name, e.returncode))
RuntimeError: 'msbuild' exited with error code: 1
scons: building terminated because of errors.
```

**Steps to reproduce:**
1. Install Mono 6.6.0 64-bit from [official site](https://www.mono-project.com/download/stable/).
2. Grab the Godot source code from [here](https://downloads.tuxfamily.org/godotengine/3.2/beta4/godot-3.2-beta4.tar.xz).
3. Compile said source code with Mono but without glue.
4. Generate the glue.
5. Recompile with Mono AND glue.
6. Catch the error which is thrown to your face by the buildsystem.

**Minimal reproduction project:**
Not needed.

**Comment by neikeq** [comment]
This was fixed by #34514

**Comment by Teashrock** [comment]
Thank you, I'll try.
